nx/test/nx/defn/compiler_test.exs

```diff
@@ -0,0 +1,49 @@
+defmodule Nx.Defn.CompilerTest do
+  use ExUnit.Case, async: true
+
+  defmodule SomeInvalidServing do
+    def init(_, _, _) do
+      :ok
+    end
+  end
+
+  test "it raises an error if the __compile__ callback is missing" do
```

Comment:
I don't believe we use the `it` convention. :)

```suggestion
  test "raises an error if the __compile__ callback is missing" do
```

Reply:
Probably autocomplete that got past by me -- I don't use it too